ownCloudAppShared/Client/Sharing/ShareViewController.swift

```diff
@@ -91,6 +91,9 @@ open class ShareViewController: CollectionViewController, SearchViewControllerDe
 	var recipientSearchController: OCRecipientSearchController?
 	var recipientsSectionDatasource: OCDataSourceComposition?
 	var recipientsSection: CollectionViewSection?
+	
+	var itemSection: CollectionViewSection?
+	var itemSectionDatasource: OCDataSourceArray?
```

Comment:
These instance variables could be moved to local variables within `init()`, since they are not used or needed anywhere else.